docs/en/dev_setup/dev_env_mac.md

```diff
@@ -72,9 +55,67 @@ To setup the environment to be able to build for Pixhawk/NuttX hardware (and ins
    sudo -H python3 -m pip install --user pyserial empty toml numpy pandas jinja2 pyyaml pyros-genmsg packaging kconfiglib future jsonschema
    ```
 
-## Gazebo Classic Simulation
+## Gazebo Simulation
+
+To setup the environment for [Gazebo](../sim_gazebo_gz/index.md) simulation with Gazebo Harmonic:
+
+1. Install Gazebo Harmonic and required dependencies:
+
+   ```sh
+   brew install gz-harmonic
+   brew install qt@5 gstreamer gst-plugins-base gst-plugins-good gst-plugins-bad gst-plugins-ugly gst-libav
```

Comment:
does the gz-harmonic formula not install its dependencies?

Reply:
Why are we doing this if `./PX4-Autopilot/Tools/setup/macos.sh --sim-tools` sets everything up?